backend/src/v5/routes/teamspaces/projects/containers/containers.js

```diff
@@ -184,6 +202,132 @@ const establishRoutes = () => {
+	 *         type: string
+	 *     responses:
+	 *       401:
+	 *         $ref: "#/components/responses/notLoggedIn"
+	 *       404:
+	 *         $ref: "#/components/responses/teamspaceNotFound"
+	 *       200:
+	 *         description: returns list of revisions
+	 *         content:
+	 *           application/json:
+	 *             schema:
+	 *               type: object
+	 *               properties:
+	 *                 revisions:
+	 *                   type: array
+	 *                   items:
+	 *                     type: object
+	 *                     properties:
+	 *                       _id:
+	 *                         type: string
+	 *                         description: Revision ID
+	 *                         example: ef0855b6-4cc7-4be1-b2d6-c032dce7806a
+	 *                       author:
+	 *                         type: string
+	 *                         description: name of the creator of the revision
+	 *                         example: someUser
+	 *                       timestamp:
+	 *                         type: string
+	 *                         description: Revision creation date
+	 *                         example: 2018-06-28T11:15:47.000Z
+	 *
+	 *
+	 */
+	router.get('/:container/revisions', hasReadAccessToContainer, getRevisions);
+
+	/**
+	 * @openapi
+	 * /teamspaces/{teamspace}/projects/{project}/containers/{container}/revisions/:revision:
+	 *   patch:
+	 *     description: Updates the void status of a revision
+	 *     tags: [Containers]
+	 *     operationId: updateRevisionStatus
+	 *     parameters:
+	 *       - teamspace:
+	 *         name: teamspace
+	 *         description: Name of teamspace
+	 *         in: path
+	 *         required: true
+	 *         schema:
+	 *           type: string
+   	 *       - project:
+	 *         name: project
+	 *         description: Project ID
+	 *         in: path
+	 *         required: true
+	 *         schema:
+	 *         type: string
+   	 *       - container:
+	 *         name: container
+	 *         description: Container ID
+	 *         in: path
+	 *         required: true
+	 *         schema:
+	 *         type: string
+	 *       - revision:
+	 *         name: revision
+	 *         description: Revision ID
+	 *         in: path
+	 *         required: true
+	 *         schema:
+	 *         type: string
+	 *     requestBody:
+	 *       content:
+	 *         'application/x-www-form-urlencoded':
+	 *           schema:
+	 *             properties:
+	 *               void:
+	 *                 description: The new status value
+	 *                 type: boolean
+	 *             required:
+	 *               - status
+	 *     responses:
+	 *       401:
+	 *         $ref: "#/components/responses/notLoggedIn"
+	 *       404:
+	 *         $ref: "#/components/responses/teamspaceNotFound"
+	 *       200:
+	 *         description: updates the status of the revision
+	 */
+	router.patch('/:container/revisions/:revision', hasReadAccessToContainer, updateRevisionStatus);
```

Comment:
2 things:
- you need write access to the container to update a revision status, so you need to write a new validator.
- you need to add a middleware to check the payload (e.g. it only has void in it and it's of type boolean)
   - maybe add it in `middleware/dataConverter/revisions` ?
   - use [yup](https://www.npmjs.com/package/yup)